components/brave_ads/browser/ads_p2a.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_P2A_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_P2A_H_
+
+#include <stdint.h>
+
+#include <string>
+
+class PrefService;
+class PrefRegistrySimple;
+
+namespace base {
+class DictionaryValue;
+}
+
+namespace brave_ads {
+
+class AdsP2A {
+ public:
+  AdsP2A();
+  ~AdsP2A();
+
+  static void RegisterPrefs(
```

Comment:
this could fit into a single line

Reply:
following style we use in `bat-native-ads`. Happy to fit in single line since this is not bat native ads